spec/integration/util/windows/security_spec.rb

```diff
@@ -388,8 +387,8 @@ def grant_everyone_full_access(path)
 
         describe "#owner=" do
           it "should accept a user sid" do
-            winsec.set_owner(sids[:admin], path)
-            expect(winsec.get_owner(path)).to eq(sids[:admin])
+            winsec.set_owner(sids[:current_user], path)
```

Comment:
I think the only potential problem here is that previously we created files with one owner, then used `set_owner` to change the owner (on dev boxes it *probably* went from `Administrator` to `Administrator`, but in Jenkins it should be from the Jenkins service account -> `Administrator`).

By leaving the user the same we're actually not quite testing the same thing.  I wonder if we can switch this to `:guest` instead of `:current_user` and continue to test the previous mutating behavior of the test.